detekt-rules/src/main/kotlin/OrtImportOrder.kt

```diff
@@ -68,7 +68,7 @@ class OrtImportOrder(config: Config) : Rule(config) {
         if (importPaths != expectedImportOrder) {
             val path = importList.containingKtFile.absolutePath()
             val message = "Imports in file '$path' are not sorted alphabetically or single blank lines are missing " +
-                "between different top-level packages"
+                "between different top-level packages."
```

Comment:
FYI, while I'm a known fan of properly ending sentences with dots, in this case the dot was deliberately omitted to align with built-in detekt rules, which suffix the message with the name of the rule in square brackets, e.g. `Needless blank line(s) [NoConsecutiveBlankLines]`.

So I'd have a slight preference for dropping this commit. But I'm also ok to keep it.

Reply:
Dropped the commit, I agree that this message should be in line with other detekt messages.